user-interface/src/lib/components/combobox/ComboBox.scss

```diff
@@ -87,6 +94,11 @@ $listItemSelectedBorderColor: #949494;
             text-decoration: none;
             width: calc(100% - 4px);
             padding: 0.5rem;
+
+            svg.usa-icon {
+              min-width: 1em;
```

Comment:
1rem? commented the same on 412, ignore that comment if resolved here

Reply:
Yeah.  Same response as above.  the 1em comes from uswds scss.